src/Microsoft.TestPlatform.TestHostProvider/Hosting/DefaultTestHostManager.cs

```diff
@@ -259,10 +261,10 @@ private int LaunchHost(TestProcessStartInfo testHostStartInfo)
             catch (OperationCanceledException ex)
             {
                 this.OnHostExited(new HostProviderEventArgs(ex.Message, -1, 0));
-                return -1;
+                throw;
```

Comment:
Is it an expectation that LaunchTestHost should throw? Why not let the caller get the failure from HostExited event?